copilot/fastapi/tests/test_kubernetes_api.py

```
@@ -0,0 +1,48 @@
+import pytest
```

Ruff (0.8.2)

`pytest` imported but unused

Remove unused import: `pytest`

(F401)